salt/utils/decorators/__init__.py

```diff
@@ -1,18 +1,20 @@
 # -*- coding: utf-8 -*-
```

Comment:
That was unintentional, thanks for catching it. I had backed up my git clones to a flash drive when my old work laptop was in the process of dying, and the flash drive was formatted FAT32, so this messed up the file permissions when I copied them back onto the EXT4 partition on my new work laptop.